server/api/field.go

```diff
@@ -32,6 +33,7 @@ type Field struct {
 	// number.
 	AutocompleteLabel string `json:"autocomplete_label"`
 	AutocompleteHint  string `json:"hint"`
+	Positional        bool   `json:"positional"`
```

Comment:
Right, so my idea of setting `Name` to $1 obviously won't work because we still need a Name for JSON. What about having a `PositionalAlias="$1"`?

3/5 we should not rely on the order of `Field`s in the array for positional args, the order is already used for Modal, and may not be a match. Explicit assignment for positional args would be more accomodating,

Reply:
@levb @larkox I'm thinking we should go with ints, as they are indexes in the "array" of positional arguments. So I'm proposing we have a `Position int` field.